2022-07_sprint/sprint_parties/index.md

```diff
@@ -1,8 +1,185 @@
 # Sprint parties
 ## Organizers
+### Data Umbrella Organizers
+
+::::{grid} 2 2 3 5
+:gutter: 3
+
+:::{grid-item-card}
+:link: https://reshamas.github.io
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Reshama Shaikh" src="../../_static/reshama.jpg" class="rounded-circle">
+
+Reshama Shaikh
+:::
+:::{grid-item-card}
+:link: https://twitter.com/BerylKanali
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Beryl Kanali" src="../../_static/beryl.JPG" class="rounded-circle">
+
+Beryl Kanali
```

Comment:
@BerylKanali is twitter ok as the link your image points to? Would you prefer some other link?

Reply:
@OriolAbril 
Is there a way to add someone's twitter under it?
So, name link to point to one of these preferred places:  website, linkedIn, github
second row to point to a second place:  twitter

Example:  https://www.dataumbrella.org/about/team

Reply:
yes, it is possible. Would the images then not be clickable? We can also use something like https://www.arviz.org/en/latest/our_team.html to include multiple links for eveyone

Reply:
I really like the ArViz one!

For the "Contributors" section, we will need one more line for location.  Example:

**Genesis Valencia
New York, NY, USA
[social links]**